animation_nodes/math/quaternion.pyx

```diff
@@ -49,6 +49,17 @@ cdef void quatFromAxisAngle(Quaternion *q, Vector3 *axis, float angle):
     q.z = axis.z * factor
     q.w = cq
 
+#https://www.euclideanspace.com/maths/geometry/rotations/conversions/quaternionToAngle/index.htm
+cdef void quatToAxisAngle(Vector3 *v, float *a, Quaternion *q):
+    quaternionNormalize_InPlace(q)
+    cdef float k = sqrt(1 - q.w * q.w)
+    
+    v.x = q.x / k
```

Comment:
When you have division like this, insert `@cython.cdivision(True)` and make sure you don't have zero division.